test/integration/freetext_editor_spec.js

```diff
@@ -2025,4 +2025,80 @@ describe("FreeText Editor", () => {
       );
     });
   });
+
+  describe("Focus must go on the current page", () => {
+    let pages;
+
+    beforeAll(async () => {
+      pages = await loadAndWait(
+        "tracemonkey.pdf",
+        ".annotationEditorLayer",
+        100,
+        async page => {
+          await page.waitForFunction("!!window.PDFViewerApplication.eventBus");
```

Comment:
Can we please wait for the entire viewer to initialize here, using the intended mechanism?
```suggestion
          await page.waitForFunction("window.PDFViewerApplication.initialized");
```

---

Ideally we'd just await `PDFViewerApplication.initializedPromise` directly, but I seem to recall Puppeteer not supporting that?